app/resources/base.html.template

```diff
@@ -102,6 +107,10 @@
     class="{{env.ui}} {% block extra_body_class %}{% endblock %}"
     {% if not env.amp %}onload="{{onload_function}}"{% endif %}>
   {% block body %}
+    {% if env.amp %}
+      <amp-analytics config="https://www.googletagmanager.com/amp.json?id=GTM-MVDGQ3H"
```

Comment:
Is this id=GTM-MVDGQ3H specific to google.org domain? If so, put the value in `config` to make it customizable.

Reply:
done